src/register_types.cpp

```diff
@@ -0,0 +1,42 @@
+#include "register_types.h"
+
+#include <godot/gdnative_interface.h>
+
+#include <core/class_db.hpp>
+#include <core/defs.hpp>
+#include <godot.hpp>
+
+#include "godot_fmod.h"
+
+using namespace godot;
+
+void initialize_fmod_module(ModuleInitializationLevel p_level) {
+	if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) {
+		return;
+	}
+
+    Callbacks::GodotFileRunner::get_singleton();
```

Comment:
Is this still needed in Godot 4?

Reply:
that call is made because it forces the singleton to be initialised here. I would leave it here as it is.